_layouts/default.html

```diff
@@ -5,7 +5,7 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
         <title>{{ page.title }}</title>
         <meta name="viewport" content="width=device-width">
-        <link rel="icon" type="image/x-icon" href="../images/logo.png"/>
+        <link rel="icon" type="image/x-icon" href="images/logo.png"/>
```

Comment:
It seems that @sivaraam suggested either `/images/logo.png` or just adding a link to it or a copy of it named `favicon.ico`. But this is doing something else. Is there a reason why you think Kaartic's suggestions will not work, or what you are doing is better?

Reply:
If I'm not mistaken, this is how it works:
```
_layouts
    L default.html
images
    L logo.png
```
And what I did then - it worked correctly, but I don't know why: `rev_news / rev_news.md` do not work. 
But here's what confused me:
https://github.com/git/git.github.io/issues/496#issuecomment-821791547

Reply:
As far as I understand, paths in the layout are referenced relatively from the path **where the layout is used**. So, your relative reference worked properly for the index and about pages but not for Rev News pages.

That's why I suggested using an absolute path instead.